packages/blaze/attrs.js

```diff
@@ -296,6 +296,26 @@ Blaze._makeAttributeHandler = function (elem, name, value) {
   } else if ((elem.tagName === 'OPTION' && name === 'selected') ||
              (elem.tagName === 'INPUT' && name === 'checked')) {
     return new BooleanHandler(name, value);
+  } else if (name === 'autofocus' || name === 'disabled' || name === 'hidden' || 
+             name === 'readonly' || name === 'required') {
+    return new BooleanHandler(name, value);
+  } else if (elem.tagName === 'FORM' && name === 'novalidate') {
+    return new BooleanHandler(name, value);
+  } else if (elem.tagName === 'IFRAME' && name === 'sandbox' && !value) {
+    // Disable sandbox mode (others values will be handled as a normal attr)
```

Comment:
Because we can have:
`<iframe sandbox>`
And this way, the attribute is simply true or false.
Or we can have this others values:
```
<iframe sandbox="allow-forms">
<iframe sandbox="allow-pointer-lock">
```